twoliter/embedded/build.Dockerfile

```diff
@@ -436,11 +438,13 @@ ENV VARIANT=${VARIANT} VERSION_ID=${VERSION_ID} BUILD_ID=${BUILD_ID}
 ARG BYPASS_SOCKET
 ARG OUTPUT_SOCKET
 ARG BUILDER_UID
+ARG KMOD_KITS
```

Comment:
While we're making this change, it'd be great to also suppress `rpm2migrations` if the in-place updates feature isn't active.